### ISSUE TITLE
Exceeding per-minute rate limit

### ISSUE THREAD

**Issue by basher83** [entropy-research/Devon#44]
It would be great if there could be some way that Devon outputs a history so that if the workflow is stopped it can be resumed again upon reloading. I Have tried a few tricks to keep my workflow going but I still keep hitting rate limits. My attempt was to create an instructions.txt with a very detailed explanation of what I wanted. In my instructions I told Devon to keep a running log in a separate txt file so that the workflow could be easily restarted in case of a crash. Within my instructions.txt I tell Devon to first read the previous log and then continue working. It kind of works, but not well enough to get me through the entire project.

**Comment by ObjectJosh** [comment]
@basher83 Yup, better persistence / restoring from a history is definitely on the list, in order to make it a lot more usable. In the meantime, a quick trick some have done just to avoid the per-minute rate limit is add a sleep duration in the code.

**Comment by basher83** [comment]
@ObjectJosh Thanks for the tip! I suppose a sleep duration would have been a simpler approach to my complex solution. I appreciate the feedback and look forward to how this project improves and evolves.
